recipe/meta.yaml

```diff
@@ -10,23 +10,23 @@ source:
   sha256: "fdab8129a1cb935db09f1832e3a7d511a4aeb2b9bb3602ca6a7ccb9730d5c9c3"
 
 build:
-  number: 1
+  number: 2
   skip: true  # [win]
 
 requirements:
   build:
     - m4
     - perl
-    - readline 6.2*
+    - readline 6.2.*
     - openssl 1.0.*
-    - ncurses
```

Comment:
Do we need to remove the build which doesn't pin ncurses from the conda-forge channel?